setup-utils/src/helpers.rs

```diff
@@ -48,8 +54,9 @@ pub fn print_hash(hash: &[u8]) {
 
 /// Multiply a large number of points by a scalar
 pub fn batch_mul<C: AffineCurve>(bases: &mut [C], coeff: &C::ScalarField) -> Result<()> {
+    let coeff = coeff.to_repr();
     let mut points: Vec<_> = cfg_iter!(bases)
-        .map(|base| base.into_projective().mul(*coeff))
+        .map(|base| base.mul_bits(BitIteratorBE::new(coeff)))
```

Comment:
This change ensures that we use mixed addition, which makes the multiplication slightly faster.